.rhcicd/pr_check.sh

```diff
@@ -33,7 +33,10 @@ export COMPONENT_NAME="notifications-backend"
 export REF_ENV="insights-stage"
 export IMAGE="quay.io/cloudservices/notifications-backend"
 export DEPLOY_TIMEOUT="1200"
-export EXTRA_DEPLOY_ARGS="--set-parameter notifications-engine/IMAGE_TAG=${IMAGE_TAG} --set-parameter notifications-connector-email/IMAGE_TAG=${IMAGE_TAG} --set-parameter notifications-connector-google-chat/IMAGE_TAG=${IMAGE_TAG} --set-parameter notifications-connector-microsoft-teams/IMAGE_TAG=${IMAGE_TAG} --set-parameter notifications-connector-servicenow/IMAGE_TAG=${IMAGE_TAG} --set-parameter notifications-connector-slack/IMAGE_TAG=${IMAGE_TAG} --set-parameter notifications-connector-splunk/IMAGE_TAG=${IMAGE_TAG} --set-parameter notifications-connector-webhook/IMAGE_TAG=${IMAGE_TAG} --no-remove-resources historical-system-profiles --no-remove-resources system-baseline"
+export EXTRA_DEPLOY_ARGS="--set-parameter notifications-engine/IMAGE_TAG=${IMAGE_TAG} --set-parameter notifications-connector-email/IMAGE_TAG=${IMAGE_TAG} --set-parameter notifications-connector-google-chat/IMAGE_TAG=${IMAGE_TAG} --set-parameter notifications-connector-microsoft-teams/IMAGE_TAG=${IMAGE_TAG} --set-parameter notifications-connector-servicenow/IMAGE_TAG=${IMAGE_TAG} --set-parameter notifications-connector-slack/IMAGE_TAG=${IMAGE_TAG} --set-parameter notifications-connector-splunk/IMAGE_TAG=${IMAGE_TAG} --set-parameter notifications-connector-webhook/IMAGE_TAG=${IMAGE_TAG}"
```

Comment:
Saving some more extra chars :) 
```suggestion
export EXTRA_DEPLOY_ARGS="""
-i notifications-engine $IMAGE_TAG
-i notifications-connector-email $IMAGE_TAG
-i notifications-connector-google-chat $IMAGE_TAG
-i notifications-connector-microsoft-teams $IMAGE_TAG
-i notifications-connector-servicenow $IMAGE_TAG
-i notifications-connector-slack $IMAGE_TAG
-i notifications-connector-splunk $IMAGE_TAG
-i notifications-connector-webhook $IMAGE_TAG
"""
```

Reply:
I love that! I have sent a patch for this one too, but with the full `--set-image-tag` parameter instead, since I like being like extra verbose with the flags to make them super clear :joy: